script.js

```diff
@@ -17,3 +22,29 @@ btnmenu.addEventListener('click', () => {
 btnclose.addEventListener('click', () => {
   item.style = 'display: none;';
 });
+
+const data = [];
+submit.addEventListener('click', (e) => {
```

Comment:
- [OPTIONAL] Following the best practice, I'll suggest selecting the form and then listening for a `submit` event instead of a `click` event on the button. eg: `const form = document.querySelector('.footer-form') form.addEventListener('submit', (e) => {
  if (emailval.val !== emailval.val.toLowerCase()) {
    e.preventDefault();
    terror.textContent = 'Error!! Please add valid email';
  }
})` :+1: :smile: